tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -58,16 +59,23 @@ def test_format_percentage(value, expected, transformer):
 @pytest.mark.parametrize(
     "unit, value, unit_length, expected",
     (
-        ("millimeter", Decimal(0.123), "short", "0.123 mm"),
+        (
+            "millimeter",
+            Decimal(0.123),
+            "short",
+            "0.1229999999999999982236431606 mm",
```

Comment:
why is this the expected result?

Reply:
its to do with Decimal(). i ran a test on it separately and that is the number it gives

Reply:
It needs to works with decimals as that's what we store for answers not floats. It would work if Decimal quantisation is True. If we provide our own format we don't need to set it to False?

Reply:
FYI, the precision issue is because Decimal is being initialised with a float, it should be `Decimal("0.123").